src/com/itacademy/eshop/product/Author.java

```diff
@@ -0,0 +1,14 @@
+package com.itacademy.eshop.product;
+
+public class Author {
+    private String author;
```

Comment:
author is a fine name for a parameter but your class is already named Author. Probably would better to have it maybe like name. Having just a single parameter for an object is redundant. You could add extra info about the author like age or gender for example